README.md

```diff
@@ -95,6 +95,8 @@ julia -e 'using Pkg; Pkg.Registry.add("General"); Pkg.Registry.add(Pkg.Registry.
 
 Additionally, `buildcell` as a part of `AIRSS` needs to be installed if one wants to use the RSS functionality:
 
+> ℹ️ To be able to build the AIRSS utilities one needs gcc and gfortran version 5 and above. Other compiler families (such as ifort) are not supported.
+
```

Comment:
Can you add an installation instruction? This would help when wanting to start fast.

Reply:
Will do , sure

Reply:
Done